homeassistant/components/entur_public_transport/sensor.py

```diff
@@ -0,0 +1,249 @@
+"""
+Real-time information about public transport departures in Norway.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.entur_public_transport/
```

Comment:
New format standard here, so keep only line 2.
`"""Real-time information about public transport departures in Norway."""`

Reply:
Fixed